docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  service_websocket:
+    build:
+      context: .
+    ports:
+      - "3000:3000"
+
+  redis:
+    image: redis:3.2-alpine
+    ports:
+      - "6379:6379"
```

Comment:
you should set a name for the network for an easier access to the redis container in other projects